src/app/routes/getInitialData/index.js

```diff
@@ -41,9 +14,13 @@ const getInitialData = async ({ match }) => {
     const response = await fetch(url);
 
     const data = await response.json();
-    const amp = isAmpPath(path);
+    const isAmp = amp === '.amp';
```

Comment:
Whilst marginally more concise, it also feels like something that might seem bizarre or require exploring more of the codebase than necessary for a newcomer. 🤔 There is very little context for the string unless you know how it's getting parsed.

Reply:
Sure, so rather than explicitly comparing the value to a string, (which means we are effectively maintaining routing in 2 places) we could just coerce the value into its bool value, i.e. `!!amp`. What do you think about that?